package.json

```diff
@@ -16,6 +16,7 @@
     "@aws-sdk/client-cloudwatch": "3.x",
     "@aws-sdk/client-codebuild": "3.x",
     "@aws-sdk/client-codepipeline": "3.x",
+    "@aws-sdk/client-dynamodb": "3.x",
```

Comment:
So what is this package.json used for?  Are you running the code for these functions locally somehow?

Reply:
Really just so that my IDE has type info